experiment/simulation/index.html

```diff
@@ -1,13 +1,78 @@
 <!DOCTYPE html>
 <html>
-<head>
-	<!-- Add CSS at the head of HTML file -->
-    <link rel="stylesheet" href="./css/main.css">
-</head>
-<body>
-		<!-- Your code goes here-->
-
-	<!-- Add JS at the bottom of HTML file --> 
-        <script src="./js/main.js"></script>
-</body>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/virtual-labs/virtual-style@0.0.6-b/css/style.min.css">
+		<link rel="stylesheet" href="./css/main.css">
+	</head>
+
+	<body class="page">
+		<div class="v-tabs">
+			<ul>
+				<li class="is-active">
+					<a href='./index.html'>
```

Comment:
Do you need to provide a link to the same page? In some of your previous experiments, you have avoided doing that. Please follow a consistent approach.
Also I think instead of creating separate pages for the 2 experiments, it may be more efficient to have 2 separate divs for them and control them through their display attributes. I am not advising to refactor this code because it'd be too much work but you can try that for the remaining experiments.